### PR TITLE
Build esp image on x86_64 arch only

### PR DESCRIPTION
**Pull request by elfosardo**

### CODE REVIEW

File: Dockerfile.ocp [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
+RUN if [ "$ARCH" == "x86_64" ]; then \
+      yum install -y gcc git make genisoimage xz-devel grub2 grub2-efi-x64 shim dosfstools mtools && \
+      dd bs=1024 count=3200 if=/dev/zero of=esp.img && \
       mkfs.msdos -F 12 -n 'ESP_IMAGE' ./esp.img && \
       mmd -i esp.img EFI && \
       mmd -i esp.img EFI/BOOT && \
       mcopy -i esp.img -v /boot/efi/EFI/BOOT/BOOTX64.EFI ::EFI/BOOT && \
       mcopy -i esp.img -v /boot/efi/EFI/redhat/grubx64.efi ::EFI/BOOT && \
-      mdir -i esp.img ::EFI/BOOT
+      mdir -i esp.img ::EFI/BOOT; \
```
Comment (juliakreger) [inline Dockerfile.ocp]: you shouldn't need  a semi-colon there....

Comment (elfosardo) [reply Dockerfile.ocp]: whoops! left it when splitting lines

Comment (elfosardo) [reply Dockerfile.ocp]: actually this is a continuation of the precedent line, so the semi-colon should be there

Comment (juliakreger) [reply Dockerfile.ocp]: ahh, I see it now. Interesting that they are &&\'ed. but *shrug*

**Pull request merged**